.ort.yml

```diff
@@ -1,6 +1,15 @@
 excludes:
+  paths:
+  - pattern: "test/**"
+    reason: "TEST_OF"
+    comment: "Directory is only used for testing. Not included in released artifacts."
+  - pattern: "@here/*/test/**"
+    reason: "TEST_OF"
+    comment: "Directory is only used for testing. Not included in released artifacts."
+  - pattern: "@here/harp-test-utils/**"
```

Comment:
We release this package -> https://www.npmjs.com/package/@here/harp-test-utils
Maybe we shouldn't, but we do.
It's internal, but it's useful :)

Reply:
Thx @zbigg, created a fix-up PR in https://github.com/heremaps/harp.gl/pull/1512